apps/src/music/MusicView.jsx

```diff
@@ -290,7 +299,11 @@ class MusicView extends React.Component {
             {
               type: 'field_dropdown',
               name: 'sound',
-              options: [['lead', 'lead'], ['bass', 'bass'], ['drum', 'drum']]
+              options: [
+                ['all/lead', 'all/lead'],
```

Comment:
can this be a constant to be used here and in line 341?
Actually it seems like the options are the same for both, somehow they could be shared (a function or constant)